commands/add/command.js

```diff
@@ -5,21 +5,24 @@ const filterable = require("@lerna/filter-options");
 /**
  * @see https://github.com/yargs/yargs/blob/master/docs/advanced.md#providing-a-command-module
  */
-exports.command = "add <pkg> [globs..]";
+exports.command = "add <pkg..>";
 
 exports.describe = "Add a single dependency to matched packages";
 
 exports.builder = yargs => {
   yargs
     .positional("pkg", {
       describe: "Package name to add as a dependency",
-      type: "string",
-    })
-    .positional("globs", {
-      describe: "Optional package directory globs to match",
       type: "array",
     })
     .options({
+      d: {
```

Comment:
This alias is too similar to the existing `-D`/`--dev` option. Unclear why it is necessary, or what precedent.